addons/mail/static/src/scss/modal.scss

```diff
@@ -0,0 +1,5 @@
+.o_mail_resend [name="message"] {
+    text-overflow: initial !important;
```

Comment:
Can't we use ``o_text_overflow`` ? Purpose would be to use standard bootstrap or odoo classes, and especially avoid custom css with "!important". Using important is like doing a rebase p=0, that's totally classless.